jobs/socorro_import_crash_data.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import logging
+import tempfile
+import urllib2
+
+from datetime import datetime as dt, timedelta, date
+from os import environ
+
+from pyspark.context import SparkContext
+from pyspark.sql.session import SparkSession
+from pyspark.sql.types import *
+
+"""
+Adapted from:
+https://github.com/mozilla-services/data-pipeline/blob/0c94d328f243338d21bae360547c300ac1b82b12/reports/socorro_import/ImportCrashData.ipynb
+
+Original source json data (not ndjson):
+s3://crashstats-telemetry-crashes-prod-us-west-2/v1/crash_report
+
+Original destination parquet data:
+s3://telemetry-parquet/socorro_crash/v2/crash_date=20190801
+
+This job now reads json from GCS, and writes parquet to GCS.
```

Comment:
Nit: there's some inconsistency in your capitalization of proper nouns and acronyms in your doc strings. I've ignored it elsewhere but this appears to the the first time you've properly capitalized them (I find the capitalization easier to read).